guides/source/configuring.md

```diff
@@ -94,7 +94,7 @@ application. Accepts a valid week day symbol (e.g. `:monday`).
 
 * `config.eager_load_paths` accepts an array of paths from which Rails will eager load on boot if cache classes is enabled. Defaults to every folder in the `app` directory of the application.
 
-* `config.enable_dependency_loading` when true, enable the autoload loading behavior even if the application is eager loaded and have `cache_classes` as true. Default to false.
+* `config.enable_dependency_loading` -- when true, enables autoload loading, even if the application is eager loaded and `config.cache_classes` is set as true. Defaults to false.
```

Comment:
Varies a little bit by each section of this guide, the initializer section uses `:` instead of a double dash, I can change to that, if you'd like :)